e2e_tests/test_browse.py

```diff
@@ -74,6 +74,46 @@ def test_browse_sort_and_filter_functionality(
         assert cols.nth(3).inner_text() == "1"
         assert cols.nth(4).inner_text() == "1"
 
+    def test_browse_sort_and_filter_functionality_with_series_filter_wildcard_character(
+        self, authenticated_page: Page
+    ):
+        authenticated_page.goto("/browse")
+        authenticated_page.get_by_label("", exact=True).nth(2).fill("1")
+        authenticated_page.get_by_role("button", name="Apply filters").click()
+
+        table = authenticated_page.locator("#tbl_result").first
+        headers = table.locator("thead th").all_text_contents()
+        rows = table.locator("tbody tr")
+        cols = 0
+        if rows.count() > 0:
+            cols = rows.nth(0).locator("td")
+
+        assert headers == [
+            "Transferring body",
+            "Series",
+            "Last record transferred",
+            "Records held",
+            "Consignments within series",
+        ]
+
+        assert rows.count() == 2
+
+        if cols.count() > 0:
```

Comment:
can we remove these conditions like in the last PR?

Reply:
condition removed as requested